### PR TITLE
{mix}[foss/2015a] smattering of libraries (REVIEW)

### PR DESCRIPTION
**Pull request by gjbex**
For foss-2015a:
* Boost
* GSL
* GMP (both latest 5.x and 6.x)
* MPFR
* MPC
* expat
* libxml2

### CODE REVIEW

**Comment by gjbex** [comment]
Test report by @gjbex
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
Linux red hat enterprise linux computenode 6.4, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.6
See https://gist.github.com/c0e6e41abb15bdb36bd8 for a full test report.

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/m/MPC/MPC-1.0.2-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ complex numbers, providing well-defined semantics for every
+ operation. At the same time, speed of operation at high precision
+ is a major design goal."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+source_urls = ['http://www.multiprecision.org/mpc/download/']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('GMP', '5.1.3'),
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/MPC/MPC-1.0.2-foss-2015a.eb]: indenting is like the others

File: easybuild/easyconfigs/m/MPC/MPC-1.0.2-foss-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                ('MPFR', '3.1.2')]
+
+runtest = 'check'
+
+sanity_check_paths = {
+    'files': ['lib/libmpc.so', 'include/mpc.h'],
+    'dirs': []
+                     }
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/MPC/MPC-1.0.2-foss-2015a.eb]: runaway bracket

**Comment by boegel** [comment]
@wpoely86: confirmation test report would be great

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 25 out of 25 (8 easyconfigs in this PR)
Linux SL 6.5, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/1bfe0b6139a7a37924e7 for a full test report.

**Comment by boegel** [comment]
works on my end, going in, thanks @gjbex and @wpoely86 for the review!

**Pull request merged**